presto-main/src/test/java/io/prestosql/operator/TestMergeOperator.java

```diff
@@ -81,7 +82,7 @@ public void setUp()
 
         taskBuffers = CacheBuilder.newBuilder().build(CacheLoader.from(TestingTaskBuffer::new));
         httpClient = new TestingHttpClient(new TestingExchangeHttpClientHandler(taskBuffers), executor);
-        exchangeClientFactory = new ExchangeClientFactory(new ExchangeClientConfig(), httpClient, executor);
+        exchangeClientFactory = new ExchangeClientFactory(new ExchangeClientConfig(), httpClient, executor, new NoOpFailureDetector());
```

Comment:
can we simulate with HeartbeatFailureDetector?

Reply:
fixed